apps/prairielearn/python/colors.py

```diff
@@ -113,7 +114,7 @@ def to_string(
             coords = serialize.get_coords(parent, fit, False, False) + [alpha_float]
 
             # See if the color value is a match, if so, return the string
-            value = tuple(alg.round_half_up(c * 255) for c in coords)
+            value = cast(ColorTuple, tuple(alg.round_half_up(c * 255) for c in coords))
```

Comment:
Basically, the typechecker can't infer the length of `coords` (and thus the length of the tuple here), so it complains. Adding a call to `cast` is basically asserting this will always be a length 4 tuple.